### PR TITLE
Khasan toronto

### LINKED ISSUE #12: khasan -checklist
[x] Make a branch
[] Work locally
[] Make a pull request
[] Collaborate
[] Merge

### PR DESCRIPTION
**Pull request by khandkerhasan**
request for review

closes #12

### CODE REVIEW

File: khasan.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#Toronto, ON
+
+- Belfast Love
```
Comment (githubteacher) [inline khasan.md]: What should I get at Belfast ❤️

Comment (davidoevans) [inline khasan.md]: i've not heard of this one

Comment (khandkerhasan) [reply khasan.md]: craft beer

**Review by davidoevans — APPROVED** [review]

File: khasan.md (hunk 1 above)
Comment (SKelly99) [reply khasan.md]: Crafty beers are hard to keep track of.

Comment (githubteacher) [reply khasan.md]: @khandkerhasan just throw a recommend craft 🍺 and I think this will be good to ⚓️

**Comment by khandkerhasan** [comment]
merge

**Pull request merged**